cdap-app-fabric/src/test/java/co/cask/cdap/internal/audit/AuditPublishTest.java

```diff
@@ -127,6 +129,11 @@ public void testPublish() throws Exception {
     }
     expectedMetadataChangeEntities.addAll(expectedCreateEntities);
 
+    // TODO (CDAP-14733): Scheduler doesn't publish CREATE audit events. Once it does, we must expect them here, too.
```

Comment:
As a side effect of this change, it fixed CDAP-14666 (profile message processor does not publish audit log). Because of that, we now publish a METADATA_CHANGE events for each schedule of the app. But due to CDAP-14733, we cannot expect CREATE events for schedules.

Reply:
Just trying to understand this better, so now there will be a METADATA_CHANGE audit message for the schedule but since we never actually publish CREATE audit events for schedules, so we will never see the CREATE audit message for schedules right? But we can still see CREATE for other entities.

Reply:
Correct, that is why in line 130, it adds all expected changes also to the expected creates. We just can't do that for schedules until 14666 is fixed